examples/index.json

```diff
@@ -2125,8 +2125,8 @@
                 "isLearnByExample": false
             },
             {
-                "name": "Receive Emails Using a Listener",
-                "url": "receive-email-using-listener",
+                "name": "Listener",
+                "url": "ftp-listener",
```

Comment:
This is related to email listener. Right? Seems like the email files are not renamed.

Reply:
Reverted the change.